### PR TITLE
[opt](inverted index) the "unicode" tokenizer can be configured to disable stop words

### PR DESCRIPTION
**Pull request by zzzxl1993**
## Proposed changes

1. properties: "parser" = "unicode", "use_stopwords" = "none" disable stop words.

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: be/src/olap/rowset/segment_v2/inverted_index_reader.cpp [hunk 1]
```diff
@@ -382,6 +378,26 @@ InvertedIndexReaderType FullTextIndexReader::type() {
     return InvertedIndexReaderType::FULLTEXT;
 }
 
+void FullTextIndexReader::setup_analyzer_lowercase(
```
Comment (xiaokang) [inline be/src/olap/rowset/segment_v2/inverted_index_reader.cpp:381]: duplicate function to writer

File: be/src/olap/rowset/segment_v2/inverted_index_reader.cpp [hunk 2]
```diff
@@ -382,6 +378,26 @@ InvertedIndexReaderType FullTextIndexReader::type() {
...
+        analyzer->set_lowercase(false);
+    }
+}
+
+void FullTextIndexReader::setup_analyzer_use_stopwords(
+        std::unique_ptr<lucene::analysis::Analyzer>& analyzer,
+        const std::map<string, string>& properties) {
+    auto stop_words = get_parser_stopwords_from_properties(properties);
+    if (stop_words == "none") {
+        analyzer->use_stopwords(false);
```
Comment (xiaokang) [inline be/src/olap/rowset/segment_v2/inverted_index_reader.cpp:397]: change true/false to string

**Comment by zzzxl1993** [comment]
run buildall

**Review by xiaokang — DISMISSED** [review]
LGTM

**Comment by zzzxl1993** [comment]
run buildall

**Comment by zzzxl1993** [comment]
run buildall

**Comment by zzzxl1993** [comment]
run buildall

**Review by xiaokang — APPROVED** [review]
LGTM

**Review by qidaye — APPROVED** [review]

**Pull request merged**